src/frontend/components/Loader/index.tsx

```diff
@@ -17,13 +17,14 @@ const Preloader = styled(Box)`
 `;
 
 interface SpinnerLookProps {
+  color?: string;
   size?: 'small' | 'medium' | 'large';
 }
 
 const SpinnerLook = styled.div<SpinnerLookProps>`
   border: 0.125rem solid transparent;
-  border-left-color: ${normalizeColor('brand', theme)};
-  border-top-color: ${normalizeColor('brand', theme)};
+  border-left-color: ${({ color }) => normalizeColor(color || 'brand', theme)};
```

Comment:
for consistency with what is done in this component can you make like line 29 ? Not destructuring props ? Or change line 29 and 35 ? As you want.